templates/page_home.html

```diff
@@ -10,76 +10,78 @@
 
     <section id="hero" class="">
 
-        <h1 class="text-6xl pb-2 leading-snug">DjangoCon Africa <br /><span class="text-lightPrimary">2025</span></h1>
+        <h1 class="text-6xl pb-2 leading-snug">{% translate "DjangoCon Africa" %} <br /><span class="text-lightPrimary">2025</span></h1>
         <!-- <h2 class="text-5xl font-bold text-tz_flag_blue-600 mb-4">Arusha, Tanzania</h2>
     <p class="text-3xl font-semibold text-gray-700 mb-2">11 - 16 August</p> -->
 
-        <p>We are in the early phases of planning the next DjangoCon Africa!</p>
-        <p>Follow our socials to stay up to date</p>
+        <p>{% translate "We are in the early phases of planning the next DjangoCon Africa!" %}</p>
```

Comment:
When running the site, this isn't translated.

![image](https://github.com/user-attachments/assets/e9ea8e17-39c8-4492-921e-08a77159e365)

Some things are translated and other things are not.

Reply:
You must be running the English version. The way translations and localization work is that you need to open the URL for each locale. For English this is `127.0.0.1:8000/en/` and 
<img width="1434" alt="Screenshot 2024-11-24 at 16 20 29" src="https://github.com/user-attachments/assets/8b25dde0-7110-4ac9-a123-979c76f0b031">

for French `127.0.0.1:8000/fr/`. 
<img width="1439" alt="Screenshot 2024-11-24 at 16 33 16" src="https://github.com/user-attachments/assets/39808a9e-e1c0-4452-8996-853a3ef79d38">

These are the only supported languages so far.

Reply:
Look at the top right of the screenshot I sent.

Maybe I'm doing something wrong. I was only able to get part of the page translated to french when I tried to run it in french.

Reply:
@sheenarbw I am not sure what you're doing wrong, but you're not supposed to translate the page to French. You're supposed to use the French locale. That's how Django works, through locales and not translations. 

The translations are done manually by people, which is why I needed @Arya-AD's help with the file [locale/fr/LC_MESSAGES/django.po](https://github.com/djangocon/2025.djangocon.africa/pull/27/files#diff-6445521761a1867bb075730cf951b8e2cc72bf15c73916ae4f2fcf0ff280f84b) where the translations are for French.
Check https://docs.djangoproject.com/en/5.1/topics/i18n/translation/, and you'll see I have done what is humanly possible to enable internationalisation. For translations for all other pages, @Arya-AD would need to help us as they are done manually. 

I also used French as the starting point because we already have a French-speaking person who did translations we couldn't use last year due to the setup of our website, which didn't allow it. I didn't pick Portuguese or Spanish or any other language because we don't have a person to translate the pages for us manually.